client/src/components/rooms/RoomsMeeting.test.tsx

```diff
@@ -35,6 +36,17 @@ describe('RoomsMeeting', () => {
   const roomCallLocator = {
     roomId: 'mockRoomId'
   };
```

Comment:
Lets add another test saying , if postcall is present in config, then RoomsMeetingExperience is called with that in the props, else it is undefined in RoomsMeetingExperience